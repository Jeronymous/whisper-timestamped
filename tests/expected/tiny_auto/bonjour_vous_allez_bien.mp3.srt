1
00:00:00,460 --> 00:00:02,260
Боже улыл!

2
00:00:02,260 --> 00:00:03,300
Таков уже на меня!

3
00:00:33,000 --> 00:00:33,800
Боже улыл!

4
00:00:34,400 --> 00:00:35,680
Эскому за меня!

