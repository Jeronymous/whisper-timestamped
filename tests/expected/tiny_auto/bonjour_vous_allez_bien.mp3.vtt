WEBVTT

00:00.460 --> 00:02.260
Боже улыл!

00:02.260 --> 00:03.300
Таков уже на меня!

00:33.000 --> 00:33.800
Боже улыл!

00:34.400 --> 00:35.680
Эскому за меня!

