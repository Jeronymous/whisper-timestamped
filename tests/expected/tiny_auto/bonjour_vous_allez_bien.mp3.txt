Боже улыл!
Таков уже на меня!
Боже улыл!
Эскому за меня!
